brave/brave-core
# Adds support for pending state

Resolves https://github.com/brave/brave-browser/issues/5426

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:
- clean profile
- enable rewards
- click verify
- create new account on uphold page (do not verify email yet)
- when redirected back make sure that you don't see any errors (verify button should still be there)
- clicking on verify wallet, add or withdraw funds should redirect you to second step on uphold page where you fill in your data. Complete this step
- verify your email
- go back to rewards page and reload page
- click on add funds and you should be redirected to uphold add funds page
- clicking on withdraw should redirect you to kyc flow

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
